### PR TITLE
[BEAM-10055] Add --region to python examples where it was missing

### PR DESCRIPTION
**Pull request by tedromer**
Add --region to python examples where it was missing.

R: @ibzib

### CODE REVIEW

File: sdks/python/apache_beam/examples/wordcount_minimal.py [hunk 1]
```diff
@@ -73,22 +73,26 @@ def run(argv=None, save_main_session=True):
...
+      # CHANGE 2/6: (OPTIONAL) Change this to DataflowRunner to
       # run your pipeline on the Google Cloud Dataflow Service.
       '--runner=DirectRunner',
-      # CHANGE 3/5: Your project ID is required in order to run your pipeline on
-      # the Google Cloud Dataflow Service.
+      # CHANGE 3/6: (OPTIONAL) Your project ID is required in order to
+      # run your pipeline on the Google Cloud Dataflow Service.
       '--project=SET_YOUR_PROJECT_ID_HERE',
-      # CHANGE 4/5: Your Google Cloud Storage path is required for staging local
+      # CHANGE 4/6: (OPTIONAL) The Google Cloud Engine region is
```
Comment (epicfaace) [inline sdks/python/apache_beam/examples/wordcount_minimal.py:88]: See https://cloud.google.com/dataflow/docs/concepts/regional-endpoints

```suggestion
      # CHANGE 4/6: (OPTIONAL) The Google Compute Engine region (such as us-central1) is
```

File: sdks/python/apache_beam/examples/fastavro_it_test.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
         --test-pipeline-options="
           --runner=TestDataflowRunner
           --project=...
+           --region=...
```
Comment (epicfaace) [inline sdks/python/apache_beam/examples/fastavro_it_test.py:31]:
```suggestion
          --region=...
```

**Comment by aaltay** [comment]
retest this please

**Comment by aaltay** [comment]
LGTM. I will merge after tests pass. Thank you @tedromer

**Comment by aaltay** [comment]
retest this please

**Comment by aaltay** [comment]
retest this please

**Pull request merged**